src/web/app/components/question-types/question-edit-answer-form/rank-options-question-edit-answer-form.component.ts

```diff
@@ -55,7 +55,9 @@ export class RankOptionsQuestionEditAnswerFormComponent
    * Assigns a Rank to the option specified by index.
```

Comment:
Note that in https://github.com/TEAMMATES/teammates/pull/9660/files#diff-190f85acfa2243ff158b4403f2d8c018R30, you also modify the response details directly.

In addition, it is better to use ngOnChanges.